cpp/src/parquet/arrow/arrow_reader_writer_test.cc

```diff
@@ -5584,5 +5592,95 @@ TEST_F(ParquetPageIndexRoundTripTest, EnablePerColumn) {
                             /*null_counts=*/{0}}));
 }
 
+class ParquetBloomFilterRoundTripTest : public ::testing::Test,
+                                        public ParquetIndexRoundTripTest {
+ public:
+  void ReadBloomFilters(int expect_num_row_groups,
+                        const std::set<int>& expect_columns_without_filter = {}) {
```

Comment:
It seems that expect_columns_without_filter is not necessary if there is only one test case.

Reply:
Will add one